File: lab02/ledteste.py
import RPi.GPIO as GPIO
import time

LED_AMARELO = 16  # Número do pino GPIO conectado ao LED

# Configuração inicial
GPIO.setmode(GPIO.BCM)
GPIO.setup(LED_AMARELO, GPIO.OUT)

# Ligar o LED
GPIO.output(LED_AMARELO, GPIO.HIGH)

# Aguardar 3 segundos
time.sleep(3)

# Desligar o LED
GPIO.output(LED_AMARELO, GPIO.LOW)

# Limpeza dos recursos utilizados
GPIO.cleanup()

File: lab02/ledteste.cpp
#include <stdio.h>
#include <wiringPi.h>

#define LED_AMARELO 16

int main(void) {
    wiringPiSetup();
    pinMode(LED_AMARELO, OUTPUT);

    while (true) {
        digitalWrite(LED_AMARELO, HIGH);  // Ligar o LED
        delay(1000);                  // Aguardar 1 segundo
        digitalWrite(LED_AMAREÇP, LOW);   // Desligar o LED
        delay(1000);                  // Aguardar 1 segundo
    }

    return 0;
}
